wire-library/wire-runtime-swift/build.gradle

```diff
@@ -19,8 +19,28 @@ dependencies {
   wire project(':wire-compiler')
 }
 
+task generateSwiftProtos(type: JavaExec) {
+  def swiftOut = 'src/main/swift/wellknowntypes'
```

Comment:
Should we be .gitignore-ing this folder? Otherwise we're not validating that the code which is committed is the same as is generated.

To put it another way, if I update the generator and forget to re-rerun this task the generated swift will be mismatched until someone fixes it.

I believe the test one is ignored.

Reply:
The generated Swift files for tests are [checked in](https://github.com/square/wire/blob/master/wire-library/wire-tests-swift/src/main/swift/FooBar.swift), so I figured we'd follow that pattern, but don't feel strongly at all. Generally I prefer not to check in generated code, although in this case since this project's main purpose is to do the generation it is nice to see any diffs that might arise.

WRT stale files, we're working on getting the Swift build into CI, which should hopefully alleviate that concern since CI would see dirty files and fail.

Reply:
[generated code in `wire-runtime-swift` is ignored. ](https://github.com/square/wire/blob/master/wire-library/wire-runtime-swift/.gitignore)

Reply:
I see lots of generated code that's [checked in](https://github.com/square/wire/blob/master/wire-library/wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos/custom_options/FooBar.kt) too though. That feels like the larger precedent, but I really don't feel strongly personally. Should probably check with #wire-discuss

Reply:
That generated code that is checked in is validated against HEAD so that CI will not pass if what is generated is mismatched from what is committed.

https://github.com/square/wire/blob/85dc2120adfdf489d66ed425f2cbad0ecbf0184a/.github/workflows/build.yml#L62-L65